Gems/ScriptCanvas/Code/Editor/Utilities/RecentAssetPath.cpp

```diff
@@ -15,6 +15,8 @@
 
 namespace ScriptCanvasEditor
 {
+    using namespace ScriptCanvas;
+
```

Comment:
Repeat: the `using namespace ScriptCanvas;` call needs to be moved to function scope or removed altogether